fork/dynamic-cdn-webpack-plugin/src/resolve-pkg.js

```diff
@@ -6,6 +8,13 @@
  * @returns {string|undefined} the full path of the module id requested
  */
 function resolve(moduleId, options) {
+	if (options.version) {
+		let scope;
+		if (moduleId.startsWith('@')) {
+			scope = moduleId.split('/')[0];
+		}
```

Comment:
What is it for ? This is not used

Reply:
oh thanks I have first use findPackages which is ask you a scope... removed

Reply:
oh thanks I have first use findPackages which is ask you a scope... removed